taverna-cwl-activity-ui/pom.xml

```diff
@@ -65,27 +60,45 @@
 		</dependency>
 		<!-- =====================================These Dependencies were added 
 			later !!! =============================================================== -->
+
 		<dependency>
-			<groupId>org.yaml</groupId>
-			<artifactId>snakeyaml</artifactId>
-			<version>1.17</version>
+			<groupId>org.apache.taverna.commonactivities</groupId>
+			<artifactId>taverna-cwl-utilities</artifactId>
+			<version>${project.version}</version>
+			<type>bundle</type>
 		</dependency>
 		<dependency>
+			<artifactId>taverna-cwl-activity</artifactId>
+			<groupId>${project.groupId}</groupId>
+			<version>${project.version}</version>
+		</dependency>
+			
+		<dependency>
 			<groupId>net.sf.taverna.t2.ui-api</groupId>
 			<artifactId>activity-palette-api</artifactId>
 			<version>${t2.ui.api.version}</version>
 		</dependency>
-
 		<dependency>
-			<artifactId>taverna-cwl-activity</artifactId>
-			<groupId>${project.groupId}</groupId>
-			<version>${project.version}</version>
+			<groupId>net.sf.taverna.t2.ui-api</groupId>
```

Comment:
These `net.sf.taverna.*` need to be updated to `org.apache.taverna.*` Taverna 3 dependencies, e.g.:

```xml
		<dependency>
			<groupId>org.apache.taverna.workbench</groupId>
			<artifactId>taverna-activity-palette-api</artifactId>
			<version>${taverna.workbench.version}</version>
		</dependency>
		<dependency>
			<groupId>org.apache.taverna.workbench</groupId>
			<artifactId>taverna-menu-api</artifactId>
			<version>${taverna.workbench.version}</version>
		</dependency>
		<dependency>
			<groupId>org.apache.taverna.workbench</groupId>
			<artifactId>taverna-contextual-views-api</artifactId>
			<version>${taverna.workbench.version}</version>
		</dependency>
		<dependency>
			<groupId>org.apache.taverna.workbench</groupId>
			<artifactId>taverna-activity-tools</artifactId>
			<version>${taverna.workbench.version}</version>
		</dependency>
```